android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -907,15 +907,25 @@ public static boolean registerAssociatedLexicalModel(String langId) {
   }
 
   public static boolean addKeyboard(Context context, HashMap<String, String> keyboardInfo) {
-    // Log Firebase analytic event.
-    Bundle params = new Bundle();
-    params.putString("packageID", keyboardInfo.get(KMManager.KMKey_PackageID));
-    params.putString("keyboardID", keyboardInfo.get(KMManager.KMKey_KeyboardID));
-    params.putString("keyboardName", keyboardInfo.get(KMManager.KMKey_KeyboardName));
-    params.putString("keyboardVersion", keyboardInfo.get(KMManager.KMKey_KeyboardVersion));
-
-    if(mFirebaseAnalytics != null) {
-      mFirebaseAnalytics.logEvent("km_add_keyboard", params);
+    String packageID = keyboardInfo.get(KMManager.KMKey_PackageID);
+    String keyboardID =  keyboardInfo.get(KMManager.KMKey_KeyboardID);
+
+    // Log Sentry analytic event, ignoring default keyboard
+    if (Sentry.isEnabled() && !(packageID.equalsIgnoreCase(KMManager.KMDefault_UndefinedPackageID) &&
+      keyboardID.equalsIgnoreCase(KMManager.KMDefault_KeyboardID))) {
+      Breadcrumb breadcrumb = new Breadcrumb();
+      breadcrumb.setMessage("KMManager.addKeyboard");
+      breadcrumb.setCategory("addKeyboard");
+      breadcrumb.setLevel(SentryLevel.INFO);
+      breadcrumb.setData("packageID", keyboardInfo.get(KMManager.KMKey_PackageID));
+      breadcrumb.setData("keyboardID", keyboardInfo.get(KMManager.KMKey_KeyboardID));
+      breadcrumb.setData("keyboardName", keyboardInfo.get(KMManager.KMKey_KeyboardName));
+      breadcrumb.setData("keyboardVersion", keyboardInfo.get(KMManager.KMKey_KeyboardVersion));
+      breadcrumb.setData("languageID", keyboardInfo.get(KMManager.KMKey_LanguageID));
+      breadcrumb.setData("languageName", keyboardInfo.get(KMManager.KMKey_LanguageName));
+
+      Sentry.addBreadcrumb(breadcrumb);
+      Sentry.captureMessage("addKeyboard", SentryLevel.INFO);
```

Comment:
If we get too many hits on this event, I may remove this and just leave the breadcrumb.

I think the breadcrumb would only get sent on a future crash

Reply:
That's probably for the best; that's exactly what they're designed for, based on my reading.  Useful info to get _on_ crashes, but unnecessary to know about if there are none.

Reply:
k. I've removed the Sentry.captureMessage() call.

I feel like we'd still like to know which keyboards users are adding. And now we only get the ones associated with crashes.

Reply:
Let's worry about analytics in a separate project. Our focus for now should be crash reporting (i.e. stability).